src/app/components/Curation/CurationPromo/index.tsx

```diff
@@ -20,6 +20,8 @@ const CurationPromo = ({
   type,
   duration: mediaDuration,
   headingLevel = 2,
+  hideImage = false,
+  hideTimestamp = false,
```

Comment:
Probably best to have the inverse - i.e showImage & showTImestamp, and both default to true